aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -81,14 +84,92 @@ def export_zip(what, outfile='testzip', overwrite=False, silent=False, use_compr
         exporting.
     :raises `~aiida.common.exceptions.LicensingException`: if any node is licensed under forbidden license.
     """
-    if not overwrite and os.path.exists(outfile):
-        raise exceptions.ArchiveExportError("the output file '{}' already exists".format(outfile))
+    if file_format is None:
+        file_format = 'zip'
+    elif file_format not in ('tar', 'zip'):
+        raise exceptions.ArchiveExportError(
+            'Can only export in either "tar" or "zip" format, please specify either for "file_format".'
+        )
+
+    filename = filename if filename is not None and isinstance(filename, str) else 'export_data.aiida'
+
+    if not overwrite and os.path.exists(filename):
+        raise exceptions.ArchiveExportError("The output file '{}' already exists".format(filename))
 
-    time_start = time.time()
-    with ZipFolder(outfile, mode='w', use_compression=use_compression) as folder:
-        export_tree(what, folder=folder, silent=silent, **kwargs)
     if not silent:
-        print('File written in {:10.3g} s.'.format(time.time() - time_start))
+        if file_format == 'tar':
```

Comment:
Note that `tar` is not the same as `tar.gz`. Tar is just an archive, no compression whatsoever, which is clearly not the case for a gzipped tarball. I really wouldn't use `tar` to actually mean `tar.gz`

Reply:
Since we do not support any other `tar` format than the g-zipped tarball, I thought it of no import. But since we are calling it the same (`tar.gz`) in the click options, I will align with that.